README.md

```diff
@@ -662,6 +662,8 @@ This will give you a local copy of the branch `daves`, and any update that has b
 <a name="i-want-to-create-a-new-remote-branch-from-current-local-one"></a>
 ### I want to create a new remote branch from current local one
 
+The following command will always push to a remote branch named origin, even if there are other remote branches with different names:
+
```

Comment:
I think "push to a remote branch named origin" is ambiguous, because origin is not the name of the branch...

I suggest for example:

The following commands will always push to the remote `origin` (even if there are other remotes with different names):

**EDIT:** as explained below I would rather propose to remove the mention of `git config push.default current`